docs/_documentations/mdt-eclipse-workingwithtemplates.md

```diff
@@ -0,0 +1,52 @@
+
+When you create a new project, you can choose from the default set of templates available in Codewind or you can choose a template source of your own. You can add more development languages and sample projects by adding new templates. 
+
+## Managing templates
+
+Template management in Codewind is provided by the **Template Source Manager**. To open the **Template Source Manager**, right-click **Local**, and then click **Manage Template Sources**. The **Template Source Manager** appears. 
+
+Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates) repository. Standard Codewind Templates, the default set of templates from Kabanero Collections,  and Appsody Stacks to develop applications with sharable technology stacks, are already included in Codewind and you can refer to those as examples.
+
+Use the **Template Source Manager** to perform the following actions:
+1. To add a new template source to the table, click `Add new`. For more information, see [Adding your own templates](#adding-your-own-templates)
+2. To remove non-default template sources, click delete. 
+3. To enable a set of templates so they appear in the **Create Project** wizard, toggle the `Enabled` slide to on. 
+4. To disable a set of templates so they do not appear in the **Create Project** wizard, toggle the `Enabled` slide to off.
+
+## Adding your own templates
+
+Add your own template to use Codewind with the framework of your choice. This is a two-step process comprising the following steps:
+1. Configure your template repository.
+2. Add your template repository to Codewind. 
+
+### Configuring your template repository
+
+To configure your template repository:
+1. Create a folder for your template in the `codewind-templates/devfiles` repository. Include the name of the framework in the name of the new folder.
+2. Add `devfile.yaml` and `meta.yaml` files.
+    - The devfile.yaml file needs to include...
+    - The meta.yaml file needs to include...
+3. Update the `codewind-templates/devfiles/index.json` file with information about your new template, for example, see [https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json](https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json).
+
+### Adding your template repository to Codewind
+
+To add your template repository to Codewind:
+1. In the Codewind view project tree, right-click **Local**. A menu appears.
+2. Click **Manage Template Sources**. The **Template Source Manager** appears.
+3. Click **Add New**.
+4. Enter the URL to your template source's index file, and press `Enter` to confirm, or `Escape` to cancel. 
```

Comment:
The user needs to fill in the URL, name and description:
![image](https://user-images.githubusercontent.com/9484320/68409435-15559680-0155-11ea-83aa-dcb9292a0239.png)

Reply:
Corrected.